docs/source/index.mdx

```diff
@@ -21,24 +25,46 @@ description: A strongly-typed, caching GraphQL client for iOS, written in Swift
   </ButtonLink>
 </p>
 
-## Benefits
+## Key Features
+
+Apollo iOS consists of three primary components, Code Generation, The Normalized Cache, and our Networking and GraphQL Execution Layer. When used together, these three components make the process of requesting, consuming, and persisting GraphQL data easier than ever before.
+
+### Code Generation
+
+Apollo iOS generates operation objects for your GraphQL operations, providing a type-safe way to construct operations and provide values for input variables in your Swift code.
 
-### Strong typing with codegen
+Each generated operation contains a set of robust, strongly-typed models for its response. These generated models make it easy to access your GraphQL response data in a type-safe and flexible format. You don't need to deal with parsing JSON responses or passing around dictionaries of values that require manual casting. You also don't need to write model types yourself, because models are generated from the GraphQL operations you define.
 
-Thanks to strong typing in Apollo iOS, you don't need to deal with parsing JSON responses or passing around dictionaries of values that require manual casting. You also don't need to write model types yourself, because models are generated from the GraphQL operations your UI defines.
+Because generated response models are operation-specific, they include properties _only_ for the GraphQL fields included in their corresponding operation. This means you can rely on the Swift type checker to flag data access errors at compile time.
 
-Because generated types are operation-specific, they include properties _only_ for the GraphQL fields included in their corresponding operation. This means you can rely on the Swift type checker to flag data access errors at compile time.
+Models for your GraphQL fragments are also generated, which operation-specific models can easily be converted to. This allows you to share common models across multiple GraphQL operations.
 
-Apollo's Xcode integration enables you to work with your UI code and corresponding GraphQL definitions side by side. It even validates your query documents, showing errors inline.
+Apollo iOS also generates models for your schema's input types, enums, and custom scalars, making working with your GraphQL data in pure Swift code safe and easy.
 
-### Normalized caching
+### The Normalized Cache
 
 Apollo iOS normalizes operation results to build a client-side cache of your data, which is updated with every operation you execute. This means your UI is always internally consistent, and it can stay up to date with your backend with as few operation as possible.
```

Comment:
```suggestion
Apollo iOS normalizes operation results to build a client-side cache of your data, which is updated with every operation you execute. This means your UI is always internally consistent, and it can stay up to date with your backend with as few operations as possible.
```